gavel/settings.py

```diff
@@ -58,6 +58,7 @@ def _list(item):
 BASE_URL =            c.get('base_url',        'BASE_URL')
 ADMIN_PASSWORD =      c.get('admin_password',  'ADMIN_PASSWORD')
 DB_URI =              c.get('db_uri',          ['DATABASE_URL', 'DB_URI'], default='postgresql://localhost/gavel')
+BROKER_URI =          c.get('broker_uri',      'BROKER_URI',               default='redis://localhost:6379/0') 
```

Comment:
When Heroku supplies Redis, it provides you the URI via `REDIS_URL`. So you should use something like what's in line 60. Adapted for BROKER_URI, it would look like this: `BROKER_URI = c.get('broker_uri', ['REDIS_URL', 'BROKER_URI'], default='redis://localhost:6379/0')`

(and then also update the comment text in `config.template.yaml` to be like line 45 in that file, specifying both options for the env var)